lib/interface.rb

```diff
@@ -0,0 +1,48 @@
+require_relative "board.rb"
+
+class Interface
+  attr_accessor :std_in, :std_out
+
+  def initialize(std_in:, std_out:)
+    @std_out = std_out
+    @std_in = std_in
+  end
+
+  def print_start
+    print(start_message)
+  end
+
+  def print_error
+    print(error_message)
+  end
+
+  def read
+    std_in.gets.chomp
+  end
+
+  private
+
+  def start_message
+    empty_board + "\n" + start_prompt + "\n" + instructions
+  end
+
+  def empty_board
+    Board.new.to_s
```

Comment:
The interface knowing about `Board`, and how to create one, stands out from the other methods of printing and getting. Might be worth injecting this in the initializer.